torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -50,13 +137,14 @@ def __init__(self, device=None, jit=False):
                                    lr=args.lr, betas=(args.adam_beta1, args.adam_beta2), weight_decay=args.adam_weight_decay,
                                    with_cuda=args.with_cuda, cuda_devices=args.cuda_devices, log_freq=args.log_freq, debug=args.debug)
 
-        INFERENCE_BATCH_SIZE = 8
+        INFERENCE_BATCH_SIZE = args.batch_size
         example_batch = next(iter(train_data_loader))
-        self.example_inputs = example_batch['bert_input'].to(self.device), example_batch['segment_label'].to(self.device)[:INFERENCE_BATCH_SIZE]
-        self.is_next = example_batch['is_next'].to(self.device)
-        self.bert_label = example_batch['bert_label'].to(self.device)
+        self.example_inputs = example_batch['bert_input'].to(self.device)[:INFERENCE_BATCH_SIZE], example_batch['segment_label'].to(self.device)[:INFERENCE_BATCH_SIZE]
```

Comment:
Sorry I don't quite understand this line, does that mean we are truncating the input to `INFERENCE_BATCH_SIZE`? What is the reason behind that?

Reply:
Yes, setting inference data to a smaller batch size. There is large training data and small inference data. We want batch size to be as small as possible because in real life it decreases latency, but needs to be bigger than 1 for sufficient utilization.

Reply:
Makes sense. I guess setting the batch size to 16 means when the batch size is 16 or larger, the job latency starts to increase.